src/main/java/api/ApiManager.java

```diff
@@ -1,8 +1,9 @@
 package api;
 
+import static io.restassured.RestAssured.given;
+
```

Comment:
Delete extra line

Reply:
It is required by checkstyle regarding static imports.